md/runtime-monitoring.md

```diff
@@ -0,0 +1,102 @@
+# Bonita Platform Monitoring
```

Comment:
⚠️ naming mismatch, we should not use platform
Platform now means 'Runtime + Studio/Dev Suite'. I know this is currently not consistent with the rest of the documentation but we should avoid adding new documentation that does not follow the Bonita glossary

Reply:
Ok, so I change all platform by **Runtime**, right?

Reply:
done